src/devices/Bmp180/Bmp180.cs

```diff
@@ -69,19 +68,19 @@ public Temperature ReadTemperature()
         public Pressure ReadPressure()
         {
             // Pressure Calculations
-            int B6 = CalculateTrueTemperature() - 4000;
-            int B62 = (B6 * B6) / 4096;
-            int X3 =  (((short)_calibrationData.B2 * B62) + ((short)_calibrationData.AC2 * B6)) / 2048;
-            int B3 = ((((short)_calibrationData.AC1 * 4 + X3) << (short)Sampling.Standard) + 2) / 4;
-            int X1 = ((short)_calibrationData.AC3 * B6 ) / 8192;
-            int X2 = ((short)_calibrationData.B1 * B62) / 65536;
-            X3 = ((X1 + X2) + 2) / 4;
-            int B4 = _calibrationData.AC4 * (X3 + 32768) / 32768;
-            uint B7 = (uint)(ReadRawPressure() - B3) * (uint)(50000 >> (short)Sampling.Standard);
-            int p = (B7 < 0x80000000) ? (int)((B7 * 2) / B4) : (int)((B7 / B4) * 2);
-            X1 = (((p * p) / 65536 ) * 3038) / 65536;
-            
-            return Pressure.FromPascal(p + ( ((((p * p) / 65536 ) * 3038) / 65536) + ((-7357 * p) / 65536) + 3791) / 8);
+            int b6 = CalculateTrueTemperature() - 4000;

```

Comment:
This will create a merge conflict with https://github.com/dotnet/iot/pull/783 I'm wondering if it won't be better to just temporarily disable all the style checks in this file and create an issue about this to avoid extra work in the other PR. cc: @MarkCiliaVincenti

Reply:
When there are conflicts, its probably easiest just to use the "own" part for now, and finally fix it again at the end. There's probably no good way of preventing conflicts with this task. And it seems it's still unclear when that PR can be merged.